src/server.c

```diff
@@ -4542,6 +4550,7 @@ sds genRedisInfoString(const char *section) {
             "loading:%d\r\n"
             "rdb_changes_since_last_save:%lld\r\n"
             "rdb_bgsave_in_progress:%d\r\n"
+            "rdb_active_cow_size:%zu\r\n"
```

Comment:
the words "rdb" should not be in that field (it represent modules and AOF too).
so either "active_child_cow_size" or "current_cow_size"?
i think "current_cow_size" is good.
```suggestion
            "current_cow_size:%zu\r\n"
```
i suppose we should rename the variable to be similar to the info field.